pcsx2/ps2/pgif.cpp

```diff
@@ -1063,7 +1052,7 @@ void dma6_OTClear()
     //u32 blkCnt = (psxHu(0x1f8010e4) >> 16) & 0xFFFF;
     u32 addrPnt = madr;
     u32 pntCnt = 0;
-    Console.WriteLn("DMA6 OT clr madr %08X  bcr %08X  chcr %08X", psxHu(0x1f8010e0), psxHu(0x1f8010e4), psxHu(0x1f8010e8));
+    pgifConLog("DMA6 OT clr madr %08X  bcr %08X  chcr %08X", psxHu(0x1f8010e0), psxHu(0x1f8010e4), psxHu(0x1f8010e8));
 
```

Comment:
Is this even related to pgif?

Reply:
idk I can clear it if you want

Reply:
Well, if it's not pgif related, there's no need to mark it as pgif, that's just confusing